output/transform/postgres_relations.go

```diff
@@ -78,6 +78,9 @@ func transformPostgresRelations(s snapshot.FullSnapshot, newState state.Persiste
 		if relation.ViewDefinition != "" {
 			info.ViewDefinition = &snapshot.NullString{Valid: true, Value: relation.ViewDefinition}
 		}
+		if relation.ToastTable != "" {
+			info.ToastTable = &snapshot.NullString{Valid: true, Value: relation.ToastTable}
```

Comment:
Why do we use `NullString` in the first place? Is it somehow a space saving versus sending an empty string?